documentation/proc-pages/physics-models/fusion_reactions/plasma_bosch_hale.md

```diff
@@ -0,0 +1,92 @@
+# Bosch-Hale Methods
+
+These methods are still kept in `physics_functions.py` but outside the [`FusionReactionRate`](plasma_reactions.md) class.
+
+## Bosch-Hale Constants | `BoschHaleConstants`
+
+The `BoschHaleConstants` class is a data structure designed to hold the constants required for the Bosch-Hale calculation for a given fusion reaction.
+
+### Attributes
+
+- **bg (float)**: Represents the Gamow energy parameter.
+- **mrc2 (float)**: Represents the reduced mass energy term.
+- **cc1 (float)**: Coefficient for the first term in the Bosch-Hale polynomial.
```

Comment:
value for these coefficients not given anywhere?

Reply:
They are in the original Bosch-Hale paper, I will link it again